Sources/Mimus/Mock.swift

```diff
@@ -91,4 +88,26 @@ public extension Mock {
 
         TestLocation.internalTestLocation = nil
     }
+
+    func record(returnValue: Any?, forCallWithIdentifier identifier: String, arguments: [Any?]? = nil) {
```

Comment:
I'd be great to have tests to cover recording `values` to `storage`

Reply:
Sure, though I'll add more of a integration tests that will verify whether recorded values are currently returned (since it matches previous patterns we had for tests).

Reply:
👍